test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,32 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() {
+        // no item
+        assertNotNull();
+
+        // only one item
+        assertNull((Object) null);
+        assertNotNull("abc");
+        assertNotNull(1);
+        assertNotNull("");
+
+        // multiple items
+        assertNull("abc", null, "ab", null);
+        assertNull(null, null);
+        assertNull(1, 2, null);
+        assertNotNull(1, Integer.valueOf(1));
+        assertNotNull("", "abc", "a", "abc");
+        assertNotNull(1, 2, 5, 8);
+    }
+
+    private void assertNull(Object... objects) {
```

Comment:
Extracting the assert function is not really necessary here since this method is quite simple (with only one line). Thus, you can add it directly into your test cases.